config/settings.py

```
@@ -50,10 +48,24 @@ def _env_list(name: str) -> list[str]:
 
 # Core
 
+ENVIRONMENTS = {"development", "production"}
+
+ENVIRONMENT = _env_str("ENVIRONMENT").strip().lower()
+
+if ENVIRONMENT not in ENVIRONMENTS:
+    raise ImproperlyConfigured(
+        f"ENVIRONMENT must be one of {sorted(ENVIRONMENTS)}, got {ENVIRONMENT!r}"
+    )
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)